packages/footer/components/Newsletter.tsx

```diff
@@ -101,6 +97,10 @@ const NewsletterForm = ({
       [subscribe]
    );
 
+   if (isSubscribed) {
```

Comment:
Why bother even passing in this prop though? Is there a reason we don't have the caller do `{!isSubscribed && <Newsletter>}` instead of returning null here?

Reply:
It can be also done that way.
However there was at least another component that had the same `return null` approach below in the same file (`NewsletterSubscribed`). 
Since this was just a hotfix I didn't changed that logic. Maybe it was done on purpose to keep the NewsletterComponent easier to read but I agree with you that we can also avoid rendering the component at all.